src/components/SearchInput/searchInput.module.css

```diff
@@ -0,0 +1,26 @@
+.container {
+  position: relative;
+  display: inline-block;
+}
+
+.searchInput {
+  padding: 0.75rem 1rem;
```

Comment:
What happen if I type the long sentence, will the icon overlap the text?

Reply:
Fixed I define maxlenght 20